api/resolvers/growth.js

```diff
@@ -125,6 +125,9 @@ export default {
     itemGrowthSubs: async (parent, { when, to, from, sub }, { models }) => {
       const range = whenRange(when, from, to)
 
+      const subExists = await models.sub.findUnique({ where: { name: sub } })
+      if (!subExists) throw new Error('Sub not found')
```

Comment:
This is noncritical nitpick. Basically if the user manually modifies the url or messes up a link they share, it will 404 now instead of showing an empty page.